apps/testnorge-inntekt/src/main/java/no/nav/registre/inntekt/utils/IdGenerator.java

```diff
@@ -0,0 +1,13 @@
+package no.nav.registre.inntekt.utils;
+
+import java.util.Random;
+
+public class IdGenerator {
```

Comment:
Jeg ville laget en private static generator inne i RsJoarkMetatdata slik at vi spiller å ha denne klassen som kun skal brukes et sted. Dette er også perfekt kode for å lage en test på.

Reply:
Litt uenig der. Blir det ikke rotete å ha hybrid-klasser? Tenkte en utility-klasse var bedre, slik at den kan brukes andre steder om det trengs?